plumber/api/importer.py

```diff
@@ -508,15 +671,40 @@ def import_vmt(
             file_system._fs,
             callbacks,
             threads,
+            # General settings (using defaults)
```

Comment:
Based on what was mentioned in the other comment, these should be named with a prefix related to what import type they belong to in the API surface